### ISSUE TITLE
Problem: The fixity task fails for rhel systems

### ISSUE THREAD

**Issue by luisilara** [artefactual-labs/ansible-role-archivematica-src#298]
There is a typo in the fixity task, it should be yum instead of apt for rhel

```
- name: "Install required rpm packages"
  apt:  <------
    name: "{{ item }}"
    state: installed
  with_items:
    - "sqlite"
    - "moreutils"
    - "mailx"
  when:
    - ansible_os_family == "RedHat"
```

https://github.com/artefactual-labs/ansible-archivematica-src/blob/d4474c3dbaef2b561c87e0650c6ee386be6910a7/tasks/fixity.yml#L13

On previous versions the issue is not present
https://github.com/artefactual-labs/ansible-archivematica-src/blob/4f97da625c005761f41508f9715c418d1935d81c/tasks/fixity.yml#L13

**Comment by amayita** [comment]
Fixed for stable/1.11.x and qa/1.x 
This issue is ready for review.

**Comment by mamedin** [comment]
Reviewed, the PR fixed the issue.
